includes/User.php

```diff
@@ -2128,15 +2109,8 @@ public function setPassword( $str, $forceLogout = true ) {
 	 * @param $str String New password to set
 	 */
 	public function setInternalPassword( $str ) {
-		$this->load();
 		$this->setToken();
 
-		if( $str === null ) {
-			// Save an invalid hash...
-			$this->mPassword = '';
-		} else {
-			$this->mPassword = self::crypt( $str );
-		}
 		$this->mNewpassword = '';
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this hard-coded password. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS2068)